modules/primer-navigation/README.md

```diff
@@ -93,12 +93,123 @@ You can also add optional headings to a menu. Feel free to use nearly any semant
 </nav>
 ```
 
+## Underline nav
+
+`.UnderlineNav` is navigation that is typically used at the top of a page as the main page navigation.
+
+```html title="UnderlineNav"
+<nav class="UnderlineNav">
+  <div class="UnderlineNav-items">
+    <a href="#url" role="tab" title="Item 1" class="UnderlineNav-item selected">Item 1</a>
+    <a href="#url" role="tab" title="Item 2" class="UnderlineNav-item">Item 2</a>
+    <a href="#url" role="tab" title="Item 3" class="UnderlineNav-item">Item 3</a>
+    <a href="#url" role="tab" title="Item 4" class="UnderlineNav-item">Item 4</a>
+  </div>
+</nav>
+```
+
+Use `.UnderlineNav-actions` to use another element alongside the underline nav
+
+```html title="UnderlineNav-actions"
+<nav class="UnderlineNav" aria-label="Foo bar">
+  <div class="UnderlineNav-items">
+    <a href="#url" class="UnderlineNav-item selected">Item 1</a>
+    <a href="#url" class="UnderlineNav-item">Item 2</a>
+    <a href="#url" class="UnderlineNav-item">Item 3</a>
+    <a href="#url" class="UnderlineNav-item">Item 4</a>
+  </div>
+  <div class="UnderlineNav-actions">
+    <a class="btn">Button</a>
+  </div>
+</nav>
+```
+
+You can right align the navigation with `.UnderlineNav--right`.
+
+```html title="UnderlineNav--right"
+<nav class="UnderlineNav UnderlineNav--right">
+  <div class="UnderlineNav-items">
+    <a href="#url" role="tab" title="Item 1" class="UnderlineNav-item selected">Item 1</a>
+    <a href="#url" role="tab" title="Item 2" class="UnderlineNav-item">Item 2</a>
+    <a href="#url" role="tab" title="Item 3" class="UnderlineNav-item">Item 3</a>
+    <a href="#url" role="tab" title="Item 4" class="UnderlineNav-item">Item 4</a>
+  </div>
+</nav>
+```
+
+This also works when adding actions.
```

Comment:
Suggest changing description to: `UnderlineNav--right` also works with when used with `UnderlineNav-actions`.